node-runner-cli/babylonnode.py

```diff
@@ -29,7 +29,7 @@
 apicli = ArgumentParser(
     description='API commands')
 api_parser = apicli.add_argument(dest="apicommand",
-                                 choices=["system", "core", "metrics"])
+                                 choices=["system", "metrics"])
```

Comment:
can you also run the command `make  cmds-reference` and update the commands reference file?